docker-compose/hive.env

```diff
@@ -0,0 +1,23 @@
+# ConnectionURL for hive-server will be overriden, see docker-compose.yml
+HIVE_SITE_CONF_javax_jdo_option_ConnectionURL=jdbc:postgresql://database:5432/metastore
+HIVE_SITE_CONF_javax_jdo_option_ConnectionDriverName=org.postgresql.Driver
+HIVE_SITE_CONF_javax_jdo_option_ConnectionUserName=hive
+HIVE_SITE_CONF_javax_jdo_option_ConnectionPassword=hive
+HIVE_SITE_CONF_datanucleus_autoCreateSchema=false
+# metastore.uris for hive-server will be overriden, see docker-compose.yml
```

Comment:
Might just be me not understanding it fully, but where are the uris overriden? Can't see anything in docker-compose that overrides it?

Reply:
sorry, it was copy-paste from [the previous configuration](https://github.com/zhenik-poc/big-data-stack-practice/blob/master/hive.env#L7). I will add hive in server mode to `docker-compose.yml` to have actually full picture how to run hive image in two different modes (metastore and hive-server)

Reply:
Aha got it!